views/build/moduleWriter.js

```diff
@@ -34,7 +34,7 @@ define([], function () {
     return function moduleWriter(moduleName, compiled) {
         let handlebars = 'handlebars';
         if (isPciRuntime(moduleName)) {
-            handlebars = 'taoQtiItem/portableLib/handlebars';
+            handlebars = 'taoQtiItem/portableLib/handlebars_4';
```

Comment:
**issue:** The idea looks very good. However, we need to make sure that `taoQtiItem/portableLib/handlebars_4` is loaded in the first place. Without this, the runtime will declare a dependency on an inexistent package...

See https://github.com/oat-sa/extension-tao-itemqti-pci/pull/379#discussion_r1343981613